src/modules/Submissions/components/List/ModalСreateSubmission/container.js

```diff
@@ -83,6 +85,16 @@ class ModalCreateSubmissionBuilder extends ContainerBuilder {
             options = files.map(({ name }) => name);
           } else {
             options = await getFileNamesFromZip(files[0]);
+            const fileName = files[0];
+            const parts = fileName.originalName.split("-");
```

Comment:
If split yields 3 fragments, we lose the 3rd one. I suggest we limit the split by 2 parts.

Reply:
Greater or equal sign in comparison with the length of the array is changed to equal sign.
Done.